### ISSUE TITLE
Is it possible to use the pandoc folding module with markdown files ?

### ISSUE THREAD

**Issue by lyndhurst** [vim-pandoc/vim-pandoc#453]
I have spent hours trying to keep my markdown files to use the markdown filetype and use the plugin's folding functionalities without any success.

I am aware of the option to keep the markdown filetype, it works fine but even with markdown and pandoc in the array, folding won't work.

I am just wondering if I am missing something or if what I am trying to do is just not possible.

Thanks in advance.

**Comment by alerque** [comment]
Did you figure this out? Care to report back or do you perhaps have dotfiles posted people could review?

**Comment by lyndhurst** [comment]
I am sorry for closing without explanation.

I tried a few different approaches like trying to set available options or setting multiple file types like `pandoc.markdown` etc... At the end, it was not very successful, and I was afraid I would have more problems in the future to maintain the config. I really like vim-pandoc folding, and vim-pandoc-syntax highlights, and I did not want to choose. Of course, there are many more great features in vim-pandoc, but for now that was my focus...

Ultimately, I was set on keeping the markdown file type for plugin compatibility but, trying to make them work with pandoc files was actually very easy. I use `zk.nvim` for notebook management, `mkdnflow.nvim` for markdown editing functionalities, and it turns out they both have options to add support for more filetypes that work very well. then it was just a matter of adding snippets which was surprisingly easy with `lua-snip` and `friendly-snippets`.

```lua
require'luasnip'.filetype_extend("pandoc", {"markdown"})
```

Thanks for taking the time to review this issue.
